crates/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -93,7 +85,7 @@ pub fn remove_packages(
     let installs_table = {
         let installs_field = toml
             .get_mut("install")
-            .ok_or(TomlEditError::MissingInstallTable)?;
+            .ok_or(TomlEditError::PackageNotFound(pkgs[0].clone()))?;
```

Comment:
suggestion (wording, non-blocking)

wouldnt `MissingInstallTable` print something like: "No packages installed"?
sounds a bit more precise than `PackageNotFound`

Reply:
just saw the tests output below, i guess for single package uninstalls this does look better with the package included 👍🏼

Reply:
I actually had a separate `NoPackagesFound` variant as part of an intermediate state of this commit and decided to just go with the existing error variant. It gets the point across that the package you want to uninstall isn't there so you go look at your manifest and see that there's _no_ packages there.